source/element-parameters.md

```diff
@@ -261,8 +295,140 @@ The components of this group are:
+  Bn3L: 3.47e1      - Length integrated normal multipole component of order 3.
+```
+The length integrated values are related to the non-integrated values via
+```{math}
+  (B_n, B_s, K_n, K_s) = L \, (B_n, B_s, K_n, K_s)
+```
+where `L` is the length of the element.
+
+For a given element, when specifying a multipole of a given order, 
+the two strength components must be of the same type.
+That is, it is not permitted for one component to be length integrated and the other not,
+as well as it is not permitted for one component to be a field and the other component to be normalized.
+However, the multipole components of different order do not have to be of the same type.
+
+When multipoles are specified for a `Bend` element, the calculation of the field is
+complicated by the curvilinear coordinate system.
+The `geometry` component switch can be used to specify how to calculate the multipole fields. 
+Possible settings for this component are:
+```{code} yaml
+vertically_pure
+horizontally_pure
+entrance_tangent
+exit_tangent
+chord_tangent
+```
+The `entrance_tangent` setting is used when the [reference curve](#s:coords) for the 
+multipole coordinate system is the straight line tangent to the entrance coordinates of the bend. 
+Similarly, the `exit_tangent` setting is used when the reference curve is the 
+straight line tangent to the exit coordinates of the bend. And the `chord_tangent` setting is used
+when the reference curve is the straight line connecting the entrance point to the
+exit point. In all these three cases, since the multipole reference curve is a straight line, 
+Eq. [](#bbmult) is valid.
+Note that for these cases, the multipole reference curve 
+is not the same as the [`branch` reference curve](#s:coords).
+
+If `geometry` is set to `vertically_pure` or `horizontally_pure`, the reference curve
+for the multipoles is the circular arc of the bend corresponding to the `branch` reference curve. 
+This is discussed in detail in [](#c:multipole.bend).
+
+%---------------------------------------------------------------------------------------------------
+(s:meta.params)=
+## MetaP Metadata Parameters
+
+The `MetaP` parameter group can be used for metadata that describes the lattice element
+but is not part of the PALS standard. Such data is necessarily program dependent.
+Example metadata could be blueprint information, information on power supply connections, etc.
+Example:
+```{code} yaml
+MetaP:
+  alias:  an_alternative_name
+  Meta: Mark 4 quadrupole
+```
+
+Components of `MetaP` are not limited to being simple strings or numbers but can be complex 
+structures. The information contained in `MetaP` should be restricted to information that 
+does not affect simulations. Custom, program specific information should be stored elsewhere.
+
+%---------------------------------------------------------------------------------------------------
+(s:patch.params)=
+## PatchP  Patch Parameters
```

Comment:
Minor comment:  The heading would be easier to read if there was a separator between the group name and the name of the heading.  For example, `PatchP:  Patch Parameters` or `PatchP - Patch Parameters`.

Reply:
Or a difference in format would also work.

Reply:
Added colon.